src/components/CourseForm/CourseForm.jsx

```diff
@@ -51,32 +57,55 @@ const InputAuthor = styled.input`
 	width: 100%;
 `;
 
-const CreateCourse = ({
+const CourseForm = ({
 	authorsForm,
 	setAuthors,
 	addAuthorToForm,
 	deleteAuthor,
 	resetForm,
 	enabledAuthors,
+	resetAuthorForm,
 }) => {
 	const [authorInput, setAuthorInput] = useState('');
 	const [duration, setDuration] = useState('');
 	const [title, setTitle] = useState('');
 	const [descr, setDescr] = useState('');
 	const history = useHistory();
 	const token = useSelector((state) => state.authentication.token);
+	const dispatch = useDispatch();
+	const { courseId } = useParams();
```

Comment:
If I specify non-existing courseId - app crushes
![image](https://user-images.githubusercontent.com/16895020/128757666-ee3d3391-fce6-49d5-b16b-2bd888512ac3.png)
I don't recommend to store error state in redux store at the same place where you store your courses

Reply:
Same error can appear in `CourseInfo` component